lib/cryptography/utils.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Kadena.Cryptography.Utils do
+  @moduledoc """
+  Documentation for `Cryptography` utils functions.
+  """
+  @type hash :: String.t()
+  @type str :: String.t()
+  @type key :: String.t()
+  @type opts :: Keyword.t()
+
+  @spec bin_to_hex(key_bin :: binary()) :: String.t()
```

Comment:
I think the return in this spec could be a Kadena.Type.Base16String

Reply:
Agree!